email-sending-concepts/domain-reputation.md

```diff
@@ -78,9 +78,9 @@ Email receivers calculate domain reputation by continuously monitoring key metri
 | Factor | Impact on Domain Reputation |
 |--------|---------------------------|
 | **Authentication Success** | Properly authenticated emails using [SPF](/email-sending-concepts/spf), [DKIM](/email-sending-concepts/dkim), and [DMARC](/email-sending-concepts/dmarc) strengthen domain reputation |
-| **Spam Complaints** | High [complaint rates](/email-sending-concepts/complaints) significantly damage domain reputation |
-| **Engagement Metrics** | Opens, clicks, and other positive interactions improve reputation |
-| **Bounce Rates** | Excessive [hard bounces](/email-sending-concepts/hard-bounce) indicate poor list hygiene and harm reputation |
+| **Spam Complaints** | High [complaint rates](/email-sending-concepts/complaints) significantly damage domain reputation. Addresses that submit spam complaints should be removed immediately. |
+| **Engagement Metrics** | Opens, clicks, and **replies** (not forwards) improve reputation |
```

Comment:
man.... no bold, and don't mention forwards... (I'm not sure about that part, but I'm sure about replies.